js/dataset/display.js

```diff
@@ -54,10 +59,10 @@ function prepare_resources() {
                 const url = $self.parent().property('url').first().attr('href');
                 const group = $Dataset.property('alternateName').value(); // This is the slug.
 
-                if (!url.startsWith(window.location.origin)
+                if (!startsWith(url, window.location.origin)
                 // TODO: temporary fix before we move all statics on the same server
                     && !url.match(/:\/\/(.[^/]+)/)[1].endsWith('data.gouv.fr')) {
```

Comment:
Yes, in this case: there is the `match` before (native on IE) which is probably hinting babel on the resulting type.
But you're right, I might be better to be homogeneous on this one.